### DISCUSSION TITLE
itemsize property for ndarray

### DISCUSSION THREAD

**Discussion by zingdle** [wjakob/nanobind#205] [Q&A] [answered]
I noticed that most properties for n-dimensional array is well defined (doc [here](https://nanobind.readthedocs.io/en/latest/api_extra.html#_CPPv4IDpEN8nanobind7ndarrayE))

However `itemsize()` and `nbytes()` are not exposed. Is this the design desicion or will it get implemented in the future? Thanks!

**Comment by wjakob** [comment, marked as answer]
Fixed here: https://github.com/wjakob/nanobind/commit/e7c6576bde820de4fdc185afb25372b53a13de90
